src/renderer/components/SettingsModal.tsx

```diff
@@ -591,6 +594,28 @@ const PythonSettings = memo(() => {
                             value={ncnnGPU}
                             onChange={setNcnnGPU}
                         />
+                        <SettingsItem
+                            description="Maximum memory to use for NCNN inference."
```

Comment:
Again, this description is incorrect since the memory budget doesn't apply to GPU mode.

Reply:
No, I'm just dumb.